scripts/src/report/report_info.py

```diff
@@ -0,0 +1,86 @@
+
+import os
+import sys
+import docker
+import logging
+import json
+import yaml
+try:
+    from yaml import CLoader as Loader, CDumper as Dumper
+except ImportError:
+    from yaml import Loader, Dumper
+
+REPORT_ANNOTATIONS = "annotations"
+REPORT_RESULTS = "results"
+REPORT_DIGESTS = "digests"
+REPORT_METADATA= "metadata"
+
+def getReportInfo(report_path,info_type,profile_type,profile_version):
+
+    docker_command = "report " + info_type + " charts/"+os.path.basename(report_path)
+
+    set_values = ""
+    if profile_type:
+        set_values = "profile.vendortype="+profile_type
```

Comment:
Avoid concatenation like this use the % operator or the string.format(). It is more efficient.

Reply:
I personally like something like this:
set_values = "profile.vendortype=%s" % profile_type

Reply:
Done.